core/benches/banking_stage.rs

```diff
@@ -88,7 +91,13 @@ fn bench_consume_buffered(bencher: &mut Bencher) {
 
         let tx = test_tx();
         let transactions = vec![tx; 4194304];
-        let batches = transactions_to_deserialized_packets(&transactions).unwrap();
```

Comment:
can we still call this from `banking_stage/consumer.rs`? Or is it different because now we're using a filter map to just ignore the ones we can't deserialize?

Reply:
yea, sure can use the one at `banking_stage/consumer.rs`, i just feel it's more readable and bit more efficient to construct packets inline here